sklearn/datasets/twenty_newsgroups.py

```diff
@@ -216,11 +216,13 @@ def fetch_20newsgroups(data_home=None, subset='train', categories=None,
 
     Returns
     -------
-    bunch : Bunch object
-        bunch.data: list, length [n_samples]
-        bunch.target: array, shape [n_samples]
-        bunch.filenames: list, length [n_classes]
-        bunch.DESCR: a description of the dataset.
+    bunch : Bunch object with the following attribute:
+        - bunch.data: list, length [n_samples]
+        - bunch.target: array, shape [n_samples]
+        - bunch.filenames: list, length [n_classes]
```

Comment:
n_samples?

Reply:
I am trying to say that is the target_names of the returned data (similarly to `fetch_20newsgroups`). Maybe n_samples is the correct word?

Reply:
@eamanu no response here?